client/styles/_canvas.scss

```diff
@@ -208,16 +208,16 @@ body #app * {
 .minimap {
   position: fixed;
   bottom: 10px;
-  right: 20px;
+  right: 50px;
 
-  width: 260px;
+  width: 170px;
   height: auto;
   max-height: 300px;
 
   cursor: pointer;
   text-align: center;
   color: $cyan;
-  font-size: 10px;
+  font-size: 14px;
```

Comment:
todo: figure out a standard font size, or more likely use the same font as the docs icon next to it